ci/build.yml

```diff
@@ -235,6 +266,54 @@ core unix frozen debug build:
     untracked: true
     expire_in: 1 week
 
+core unix frozen R debug build:
+  stage: build
+  <<: *gitlab_caching
+  needs: []
+  variables:
+    PYOPT: "0"
+    TREZOR_MODEL: "R"
+  script:
+    - nix-shell --run "poetry run make -C core build_unix_frozen"
+  artifacts:
+    name: "$CI_JOB_NAME-$CI_COMMIT_SHORT_SHA"
+    untracked: true
+    expire_in: 10 weeks
+
+core unix frozen R debug build arm:
+  image: nixos/nix
+  stage: build
+  <<: *gitlab_caching
+  needs: []
+  variables:
+    PYOPT: "0"
+    TREZOR_MODEL: "R"
+  script:
+    - nix-shell --run "poetry run make -C core build_unix_frozen"
+    - mv core/build/unix/trezor-emu-core core/build/unix/trezor-emu-core-arm
+  artifacts:
+    name: "$CI_JOB_NAME-$CI_COMMIT_SHORT_SHA"
+    untracked: true
+    expire_in: 10 weeks
+  tags:
+    - docker_darwin_arm
+
+# Debugger build for gdb/lldb.
+core unix R debugger build:
```

Comment:
I thought it would be, but probably not - same as with GDB scripts - good just for one-time debugging. Deleted in [82e856b](https://github.com/trezor/trezor-firmware/pull/2610/commits/82e856bc54fb92fee04aee302f2832ed6725324e)